ipfx/feature_record.py

```diff
@@ -0,0 +1,136 @@
+import pandas as pd
+import numpy as np
+
+
+def build_cell_feature_record(cell_features, sweep_features):
```

Comment:
This function is doing multiple things. You should split the work.

Reply:
I refactored a bit to reduce repetition.

Reply:
@nikahk Thanx for pointing out the "multiple thing" thing. I have refactored to stay close to single responsibility.